app/controllers/api/payment/braintree_controller.rb

```diff
@@ -29,15 +29,24 @@ def webhook
     render json: {success: true}
   end
 
+  def delete_payment_method
+    # TODO: Delete payment method on Vault, and if that's successful, destroy it locally.
```

Comment:
I added this endpoint because it was going to be implemented for the member management sprint. I never finished it since we decided to pivot to GoCardless instead. Its implementation would be finished once we actually continue with the member management sprint, at which point it'll be modified to delete them on Braintree.